src/lib/components/Listeners.svelte

```diff
@@ -124,7 +128,7 @@
     </div>
 
     <div class="w-3/5 h-full bg-stone-600 p-2 overflow-y-auto scrollbar pb-28">
-      <span class="text-white"
+      <span class="text-white" style="color: #f8f8f2"
```

Comment:
since I have used `tailwindcss` you'd better change the `style` to tailwind class